tests/conftest.py

```diff
@@ -146,7 +195,19 @@ def global_execution_env():
             objectives.append(objective)
 
         assets = _TestAssets(datasets=datasets, objectives=objectives)
-        yield f, assets, n
+        return assets
+
+
+@pytest.fixture
+def data_envs_1(data_envs, client_1):
```

Comment:
Should be `data_env_1` (no plural)? Data environment for node 1.

Reply:
Yes, I forgot to rename it, thank you!